elastic/kibana
# [Infra] Fix Docker disk charts: Change disk with diskio

Closes #184038 

## Summary

This PR  fixes Docker disk charts by changing `disk` with `diskio`.

## Testing
- Collect docker container metrics (for example using metricbeat) 
   - Note: The new Container view flag in Infra settings should be disabled 
- Open Inventory and switch the `Show` setting to Docker Containers
- Open the details page using the metrics link
- The disk charts should be visible: 
![image](https://github.com/elastic/kibana/assets/14139027/5b8690c9-e4aa-4fb3-893f-28c77064ba15)
